python_basics.py

```diff
@@ -6,23 +6,24 @@
 file = "/Users/mohammadanas/Desktop/Duke MIDS/Spring 2021/SoftwareTools/Assignment_1/python-basics-biostats-821/example.txt"
 
 
-def get_data(file: list[int]) -> list[int]:
+def get_data(file: str) -> list[list[int]]:
     """This function return the data as a list of lists"""
-
     integers = []
     with open(file) as file:
         for line in file:
             l = line.split()
             integers.append(l)
         file.close()
-    return integers
+    lst1 = [int(i) for i in integers[0]]
+    lst2 = [int(i) for i in integers[1]]
+    return lst1 + lst2
 
 
-def average(lst):
+def average(lst: list[int]) -> int:
     return sum(lst) / len(lst)
 
 
-def standard_dev(lst):
+def standard_dev(lst: list[int]) -> int:

```

Comment:
Our average and standard deviation functions should return a float and not integers

Reply:
so we should adjust type hints to float

Reply:
good point. Also, the same holds true for covariance and correlation so we can change that too